.bundle/config

```diff
@@ -2,3 +2,4 @@
 BUNDLE_PATH: "vendor/bundle"
 BUNDLE_RETRY: "3"
 BUNDLE_JOBS: "3"
+BUNDLE_WITHOUT: "screenshots"
```

Comment:
_Unfortunately, I can't leave this comment inline in the file, because it gets cleaned every time `bundle install` is called with some different option._

The `fastlane/Pluginfile` contains RMagick in an optional group called screenshots.

In order to run the screenshots composition, you need to `bundle install --with screenshots`, but that will add a `BUNDLE_WITH: "screenshots"` to the `.bundle/config` file.

If that change is committed, CI builds on CircleCI will fail, because the RMagick gem installation fails to build its native dependencies.

I'm not sure how to work around this yet, but know for a fact that WooCommerce iOS has somehow done it, as their builds pass. So, while I'm gathering more info, I'm just making we don't try to install RMagick.

Yes, it's sort of pointless to have a config to exclude an already optional group. My intent is to leave a trail in the repo for future me to remember.

Reply:
I'm also running into a problem with RMagick when I try to test the PR locally, probably because I've never done our screenshot composition locally and don't already have RMagick installed.

When I run `BUNDLE_WITH=screenshots bundle install`, or when I try to install the RMagick gem directly, it fails with this error:

> ERROR: Can't install RMagick 3.2.0. Can't find ImageMagick with pkg-config

Not sure if that's the same failure you were seeing with CI builds, but I'll see if I can at least find a local solution for my error. :)

Reply:
Actually, it looks like installing RMagick has been troublesome for both local installs and on CI, and is optional in both WooCommerce and WordPress iOS. I noticed that we made it optional in the release toolkit, as well: https://github.com/wordpress-mobile/release-toolkit/pull/60

My understanding is that in our other projects we are generating the raw screenshots but not doing the composition on CI, but perhaps @loremattei can check me on all of this. :)

Reply:
Yeah! So, installing `RMagick` requires to pre-install `image-magick` via `brew`, and it can still cause some issues sometimes. We didn't want this to be a problem for the developers, so we made it optional. 
Installing `image-magick` on CI is usually very simple, though I never dug into `RMagick` on CI. 

Please, note that running this on CI is not a requirement right now. We don't expect to update the screenshots a lot...

About `.bundle/config`, I actually think we should just remove it from the repository. We don't check it in git in other project, as `rake dependencies` takes care of configuring `bundle` properly. 

I'll open a PR to remove it :-)

Reply:
🤔 I'm of two minds about removing the `.bundle/config`. I like how you can track stuff like the concurrency in there, but, at the same time, the fact that it tracks changes when we do `--with screenshots` makes it annoying to track.

[Heroku recommends not to track it](https://devcenter.heroku.com/articles/bundler-configuration), for example.

If we used it only for the concurrency and location maybe it would make sense to track it, but I can see how the `BUNDLE_WITH` and/or `BUNDLE_WITHOUT` business might make it annoying.

Reply:
@rachelmcr [this workflow](https://github.com/Automattic/simplenote-ios/blob/9a39c1d4529d49b0b3bec32d6fc6c6f3a3b23bd5/.github/workflows/screenshots.yml#L57-L62) does the composition in CI as well. The same code should work locally, it does for me at least.

Reply:
Ah yeah, thanks for that — I mixed up what exactly I was looking at and that got things working for me locally. 👍

Reply:
The PR to remove `.bundle/config` has been merged in `develop`. :-)